### PR TITLE
Add issue forms 

### PR DESCRIPTION
**Pull request by LtDan33**
Addresses #255. Added some issue forms in order to help guide issues coming in. Starting with bugs and feature requests. 

Thanks to @[antocuni](https://github.com/antocuni) for the assist.

The following issue types were suggested in the original issue created. I'm not sure if making all of them in this initial ticket makes sense.
- [x] Help requests/questions
- [x] Bug report
- [x] Feature request
- [x] Misc

**PR notes**: Due to the older way of creating issue templates being in place in another PR in parallel, I rebased to be able to take those markdown files into this PR and then remove them. Otherwise, there would be two selections for bugs using yml (forms - newer) and markdown (templates - older). You can disregard changes other than the "ISSUE_TEMPLATE" folder.

Edit:

The following are intentially left out as to not create too many issues in the beginning. If the "misc" gets too big then we can separate out as needed.
- [ ] Documentation issue
- [ ] Hosting and infrastructure
- [ ] Governance
- [ ] Security (but make that a link to https://pyscript.net/security

### CODE REVIEW

**Comment by LtDan33** [comment]
There are a few other types of issues that I didn't add as I want to see which makes sense to add in the beginning. Looking for feedback on what other ones are valuable without having too many. Also I think having a "miscellaneous" ticket for things that people don't think fit cleanly. I think it would be better to have them in that bucket than in a place it really doesn't belong in.

**Review by antocuni — COMMENTED** [review]
thank you the PR, I like it and I think that forms work better than the current templates.
In the meantime, @fpliger already merged my #252 so maybe he also wants to have a look to this.
Also, I don't know how the two interacts: what happens if you have both an issue templates and and issue forms? Maybe the best is to rebase this PR on top of the current master and delete the issue templates files.

Finally, an idea which just came to my mind: what about a new category called "Question", which we can use to redirect people to the nucleus forum? Something like "github issues are for bugs and feature requests, but you are welcome to ask your question there".
Hopefully this will help to avoid cluttering the issues with general questions as it is happening now

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+        - Other
```
[the remark this answers is not in the document]
Comment (LtDan33) [reply .github/ISSUE_TEMPLATE/bug-report.yml:47]: Good call, I can actually just make this an optional field and make a note that its not needed.

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        ### Lets make sure you are in the right place. If you have an idea/request for:
+
+        - #### A specific package/library (such as pandas or scikit learn):
+
+          Search for that respective library on github repo or website. You will have much more success there.
+
+        - #### A general Python question:
+
+          Try out a forum post here [<TBD>](python.org)
+
```
[the remark this answers is not in the document]
Comment (LtDan33) [reply .github/ISSUE_TEMPLATE/feature-request.yml:27]: Good idea.

[the remark this answers is not in the document]
Comment (kevingoldsmith) [reply .github/ISSUE_TEMPLATE/feature-request.yml:27]: general python questions should probably go to https://discuss.python.org/c/users/

**Comment by LtDan33** [comment]
Updated the top-level description, but I rebased to be able to remove the other markdown issues. Otherwise, there would be duplicate issue types until another PR corrected that. This way it should be as expected once this is merged.

**Comment by antocuni** [comment]
@LtDan33 LGTM, thank you. Sorry it took so long to review, merging it now!

**Comment by antocuni** [comment]
> merging it now!

I take it back, I just noticed that `pre-commit.ci` is failing, trying to fix it

**Comment by pww217** [comment]
pre-commit seems to have an issue with the double `::` near the top of the template. I'm not sure if that's intentionally part of the template here but it considers it a linting problem.

This is a common problem with adding too many things to pre-commit is that we use their default configurations and those are often painfully strict. You CAN merge this without the tests passing, but it'll probably bother us again at some future point. It probably was a bit rushed to add so many checks with default configs but that's where we are now.

**Review by fpliger — APPROVED** [review]

**Review by antocuni — APPROVED** [review]

**Comment by antocuni** [comment]
> GitHub still sees you as requesting a change to merge the PR even thought the actual review shows everything as resolved. 
yes, I don't know what happened. I added a new empty review as "approve" and it's merge-able now.
Also, I pushed a small commit which adds the `needs-triage` label to all new issues, as it was with the old `.md` files. I tested the new files on a test repo and they seems to work as intended. Merging the PR, this time for real :).
Thank you!

**Pull request merged**